utils/src/main/kotlin/FileMatcher.kt

```diff
@@ -28,9 +28,11 @@ import java.nio.file.Paths
  */
 class FileMatcher(
     /**
-     * The list of file names to consider for matching.
+     * The list of [glob patterns][1] to consider for matching.
+     *
+     * [1]: https://docs.oracle.com/en/java/javase/12/docs/api/java.base/java/nio/file/FileSystem.html#getPathMatcher(java.lang.String)
```

Comment:
In `Configuration.md` we link https://docs.oracle.com/javase/tutorial/essential/io/fileOps.html#glob instead, should we use it here, too? If not, we should probably still not use a Java 12 link, because we actually use Java 8 as a compile target.

Reply:
Done.